bids-validator/bids_validator/rules/session_level_rules.json

```
@@ -93,7 +95,9 @@
         "_electrodes.json",
         "_eeg.json",
         "_coordsystem.json",
-        "_photo.jpg"
+        "_photo.jpg",
+        "_photo.png",
+        "_photo.tif"
```

```suggestion
        "_photo.jpg"
```
Doesn't apply to "eeg".